lib/benchee/utility/file_creation.ex

```diff
@@ -122,10 +127,13 @@ defmodule Benchee.Utility.FileCreation do
   end
 
   defp to_filename(name_string) do
-    no_input = Benchmark.no_input
+    no_input = Benchmark.no_input()
+
     case name_string do
-      ^no_input -> ""
-      _         ->
+      ^no_input ->
+        ""
+
+      _ ->
```

Comment:
ehhh - what does the empt line above do there? o_O

Reply:
@devonestes any idea why an empty line is added in the above?

Reply:
If one of the clauses is multi-line, they all become multi-line. And all multi-line clauses get a space between them, in anonymous functions, case statements, etc.

Reply:
A LINE BETWEEN THEM THIS IS PREPOSTEROUS